### ISSUE TITLE
add all metrics at notebook

### ISSUE THREAD

**Issue by tahalpara** [sapientml/sapientml#72]
**Is your feature request related to a problem? Please describe.**
AsIs:

```
#EVALUATION
if set(_TARGET_COLUMNS).issubset(__test_dataset.columns.tolist()):
    from sklearn.metrics import mean_squared_error
    __rmse = mean_squared_error(__target_test, __y_pred, squared=False)
    print('RESULT: RMSE:', str(__rmse))

```
**Describe the solution you'd like**
This is useful for users when the script and notebook(final_script.py and final_script.py.ipynb) display all metrics instead of just one.

**Comment by arima-tsukasa** [comment]
Change final_script.py and final_script.py.ipynb to display the metrics that can be output.
It is as follows.
* classification:
  * F1
  * AUC
  * Accuracy
  * Gini
  * LogLoss
  * ROC_AUC
  * MCC
  * QWK
* regression:
  * R2
  * RMSLE
  * RMSE
  * MAE

However, Map@K is displayed only when specified.
